Build/Scripts/runTests.sh

```diff
@@ -491,12 +493,12 @@ case ${TEST_SUITE} in
         cleanTestFiles
         ;;
     composer)
-        COMMAND="composer $@"
+        COMMAND="composer $*"
         ${CONTAINER_BIN} run ${CONTAINER_COMMON_PARAMS} --name composer-command-${SUFFIX} -e COMPOSER_CACHE_DIR=.cache/composer -e COMPOSER_ROOT_VERSION=${COMPOSER_ROOT_VERSION} ${IMAGE_PHP} /bin/sh -c "${COMMAND}"
         SUITE_EXIT_CODE=$?
         ;;
     composerInstall)
-        COMMAND="composer install $@"
+        COMMAND="composer install $*"
```

Comment:
This breaks:

```bash
./Build/Scripts/runTests.sh -s composerInstall -W
```

Reply:
It broke before and it breaks with TYPO3 core‘s `COMMAND=(composer install "$@")` solution. :shrug: